.github/pull_request_template.md

```diff
@@ -1,14 +1,20 @@
 ## Pull Request Description
-
 [description here]
 
+### Related Pull Requests
+[related PRs from different repositories]
+
+### Related Issues
+[What issue(s) is the PR addressing]
+
 ## Checklist
 
 Not all may apply:
```

Comment:
Maybe we could remove "Not all may apply" and "this box must be checked..." in favor of "[Optional]" and "[Required"] item prefixes?

For example:

Optional
* Tests

Required
* Documentation
* Changelog